common/src/main/kotlin/de/hpi/cloud/common/utils/Instant.kt

```diff
@@ -0,0 +1,6 @@
+package de.hpi.cloud.common.utils
+
+import java.time.Instant
+import java.time.format.DateTimeFormatter
+
+fun Instant.toIsoString() = DateTimeFormatter.ISO_INSTANT.format(this)
```

Comment:
We can rename this file to `DateTime`  so it can also contain other related extensions